plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/connection/DockerConnection.java

```diff
@@ -39,6 +42,25 @@ public DockerConnection path(String path) {
         return this;
     }
 
+    public DockerConnection query(String name, Object... values) {
+        if (name == null) {
+            throw new IllegalArgumentException("Name is null");
```

Comment:
<code>NullPointerException</code> must be thrown in this case, see _60 Item_ from _Effective Java 2nd ed_.
Basically it says:
>If a caller passes null in some parameter for which null values are prohibited, convention dictates that NullPointerException be thrown rather than IllegalArgumentException.